virtualization/windowscontainers/management/container_networking.md

```diff
@@ -129,6 +129,8 @@ To use the Transparent networking mode, create a container network with driver n
 ```none
 C:\> docker network create -d transparent MyTransparentNetwork
 ```
+> If you encounter an error in creating the transparent network, it is possible that there is an external vSwitch on your system which is undiscoverable to Docker, and which is preventing the transparent network from being bound to your host's external network adapter. Reference the below section, 'Existing vSwitch Blocking Transparent Network Creation,' under 'Caveats and Gotchas' for more information.
```

Comment:
Make this as a Note (whatever format/syntax is used)
I would change, "undiscoverable" to "not automatically discoverable"